packages/manager/modules/pci/src/projects/project/quota/quota.service.js

```diff
@@ -9,6 +9,10 @@ export default class {
     this.OvhApiMe = OvhApiMe;
   }
 
+  /**
+   * WARNING: should not be used anymore!!!
+   * Need to use ovhPaymentMethod.getDefaultPaymentMethod instead.
+   */
```

Comment:
https://jsdoc.app/tags-deprecated.html ?

Reply:
Moreover, does this log a message to be aware of it while developing ?

Reply:
Ah, good idea